nature.md

```diff
@@ -0,0 +1,1441 @@
+<td>
+Most of the pages on Nature.com have unique page titles that describe the purpose of the page. 
+<br><br>
+Some pages (e.g in checkout flows) do not have unique page titles at every stage in a process. We are actively working to find and repair these problems. 
+</td>
+</tr>
+<tr>
+<td><a href="http://www.w3.org/TR/WCAG20/#navigation-mechanisms-focus-order"><strong>2.4.3 Focus Order</strong></a> (Level A)
+<br><br>Also applies to:<br><br>
+<em>EN 301 549 Criteria</em>
+<ul>
+<li>9.2.22 (Web)</li>
+<li>10.2.22 (non-web document)</li>
+<li>11.2.1.22 (Software)</li>
+<li>11.2.2.22 (Closed Software) – Does not apply</li>
+<li>11.6.2 (Authoring Tool)</li>
+<li>12.1.2 (Product Docs)</li>
+<li>12.2.4 (Support Docs)</li>
+</ul>
+<em>Revised Section 508</em>
+<ul>
+<li>501 (Web)(Software)</li>
+<li>504.2 (Authoring Tool)</li>
+<li>602.3 (Support Docs)</li>
+</ul></td>
+<td>Partially Supports</td>
+<td>
+The meaning or operation of the pages on Nature.com is generally unaffected by focus order. The focus order of all page content matches the visual order.
+<br><br>
+Third party scripts including advertising scripts, the Disqus commenting widget, or survey or customer services widgets (among others), may inject content where the focus order does not match the visual order. The operation and meaning of these components should be unaffected by the focus order. 
+</td>
+</tr>
+<tr id="link-purpose-in-context">
+<td><a href="http://www.w3.org/TR/WCAG20/#navigation-mechanisms-refs"><strong>2.4.4 Link Purpose (In Context)</strong></a> (Level A)
+<br><br>Also applies to:<br><br>
+<em>EN 301 549 Criteria</em>
+<ul>
+<li>9.2.23 (Web)</li>
+<li>10.2.23 (non-web document)</li>
+<li>11.2.1.23 (Software)</li>
+<li>11.2.2.23 (Closed Software) – Does not apply</li>
+<li>11.6.2 (Authoring Tool)</li>
+<li>12.1.2 (Product Docs)</li>
+<li>12.2.4 (Support Docs)</li>
+</ul>
+<em>Revised Section 508</em>
+<ul>
+<li>501 (Web)(Software)</li>
+<li>504.2 (Authoring Tool)</li>
+<li>602.3 (Support Docs)</li>
+</ul></td>
+<td>Partially Supports</td>
+<td>
+Generally, the purpose of each link that we provide can be determined by its descriptive text label; linked images or icons have alternative text or similar to provide this context. 
+<br><br>
+In research articles, anchor links that refer to same-page supplementary figures or tables may have insufficient context in their link text. Repetitive links (e.g. "show more" or "view all" type links in lists of content) may not be differentiated from each other. In limited cases we may support poor link text with the use of ARIA labels. We are actively working to repair these problems. 
+<br><br>
+Third party scripts including advertising scripts, the Disqus commenting widget, or survey or customer services widgets (among others), may inject content that includes links with no accessible text, or poorly-named accessible text. Where we know of deficiencies, we have contacted third party suppliers to request repairs.
```

Comment:
Do you think we should always use the same third party list?
When we mention 3rd parties throughout the document we sometimes mention disqus, sometimes ads, sometimes others.

Reply:
IMO, no. For each criterion, I've tried to only mention the third party scripts that we either know to have problems in that area, or that I think have the real potential to create problems. 

There will always be unknowns when it comes to third parties, but I'm wary of hedging too much. It's true that third parties can potentially inject all kinds of problematic content, but we'd want to deal with that as it arises. In the meantime our customers need to make decisions on whether or not they can buy our software based on this document, so the main focus needs to be accuracy (both from the customer's perspective and from our business' perspective).